src/config.ts

```diff
@@ -1,10 +1,13 @@
 export const config = () => ({
   port: 3000,
   chromeBinaryPath: process.env.CHROME_BINARY_PATH ?? '/usr/bin/google-chrome',
-  isTestEnvironment: process.env.NODE_ENV === Environment.TEST,
+  rejectFileProtocolUrls: !!(parseInt(
```

Comment:
NIT but we tend to use explicit Boolean casting (via `Boolean()`) instead of `!!`. That I would like to change.

And personally i like Number better because it's simpler and perfectly suitable for that case, but whatever, `parseInt` is OK too.

Reply:
I'm used to make config a function as environmental values are often read at runtime via `dotenv.config()` or something like that

Reply:
But if thats not the case, we can make it an object

Reply:
I tought about dotenv, but it seems to be out of pull request's scope.